src/test/kotlin/com/example/pass_manager/PassServiceTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.pass_manager
+
+import ClientFixture
+import PassFixture
+import com.example.pass_manager.repositories.PassRepository
+import com.example.pass_manager.service.ClientService
+import com.example.pass_manager.service.PassServiceImpl
+import com.example.pass_manager.web.dto.PriceDistribution
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.justRun
+import io.mockk.verify
+import io.mockk.verifyOrder
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import java.math.BigDecimal
+
+@ExtendWith(MockKExtension::class)
+class PassServiceTest {
+    @MockK
+    private lateinit var passRepository: PassRepository
+
+    @MockK
+    private lateinit var clientService: ClientService
+
+    @InjectMockKs
+    private lateinit var passService: PassServiceImpl
+
+    private val clientFixture = ClientFixture
+    private val passFixture = PassFixture
+
+    @Test
+    fun `calculating price distribution for client should return valid result list`() {
+        every { passRepository.findAllByClientId(any()) } returns passFixture.passes
+
+        val priceDistributions = passService.calculatePriceDistribution(clientFixture.clientId)
+        val expected = listOf("First type", "Second type", "Third type").map { PriceDistribution(it, BigDecimal.TEN) }
+
+        assertThat(priceDistributions).isEqualTo(expected)
+
+        verify { passRepository.findAllByClientId(any()) }
```

Comment:
```suggestion
        // GIVEN
        every { passRepository.findAllByClientId(any()) } returns passFixture.passes

        // WHEN
        val priceDistributions = passService.calculatePriceDistribution(clientFixture.clientId)
        
        // THEN
        val expected = listOf("First type", "Second type", "Third type").map { PriceDistribution(it, BigDecimal.TEN) }
        assertThat(priceDistributions).isEqualTo(expected)
        
        // AND THEN
        verify { passRepository.findAllByClientId(any()) }
```

Use `GIVEN-WHEN-THEN` comments, so your tests will be more readable